src/tracing/session.js

```diff
@@ -51,5 +54,13 @@ export class Session {
     }
     return this;
   }
-}
 
+  get attributes() {
+    return this.#attributes;
+  }
+
+  setAttributes(attributes) {
```

Comment:
Nit: I think you can do `set attributes` here too, instead of making a function.

##### non-blocking

Reply:
The semantics are different. This method doesn't replace `attributes`, so I don't want code that looks like `attributes =`.

I'm considering `updateAttributes`.